.github/workflows/publish-rc.yml

```diff
@@ -155,3 +155,18 @@ jobs:
       # Display history to know byte size of each layer
       # Image is available only because of the previous `docker run` command
       - run: docker image history chainsafe/lodestar:${{ needs.tag.outputs.tag }}
+
+      - name: Build and push custom Grafana
+        run: >
+          docker buildx build ./docker/grafana/ --push
+          --file ./docker/grafana/Dockerfile
+          --build-context dashboards=./dashboards
+          --tag chainsafe/grafana:${{ needs.tag.outputs.tag }}
```

Comment:
needs appropriate repo created in chainsafe dockerhub cc @philknows

Reply:
@wemeetagain has access to create this repo. I'm going to advocate for a prefix like `lodestar-grafana` due to the high potential of others in the ChainSafe namespace using Grafana for their own needs and to minimize any potential confusion.

Reply:
I have renamed the images to have `lodestar-` prefix.